File: b572(forgetful_jack)/C++/forgetful_jack.cpp
#include <iostream>

int main(int argc, char const *argv[]) {
  int num_of_cases, now_hour, now_min, arr_hour, arr_min, cost_min, least_min;
  while (std::cin >> num_of_cases) {
    for (int c = 0; c < num_of_cases; c++) {
      std::cin >> now_hour >> now_min >> arr_hour >> arr_min >> cost_min;
      least_min = 60 * (arr_hour - now_hour) + arr_min - now_min;
      if (least_min >= cost_min) std::cout << "Yes" << '\n';
      else std::cout << "No" << '\n';
    }
  }
  return 0;
}

File: b572(forgetful_jack)/Python/forgetful_jack.py
from sys import stdin
while True:
    num_of_cases = stdin.readline()
    if not num_of_cases: break
    for case in range(int(num_of_cases)):
        now_hour, now_min, arr_hour, arr_min, cost_min = map(int, stdin.readline().split())
        least_min = 60 * (arr_hour - now_hour) + arr_min - now_min
        if least_min >= cost_min: print('Yes')
        else: print('No')
